modules/sugar/src/main/ts/ephox/sugar/api/dom/Truncate.ts

```diff
@@ -1,10 +1,15 @@
 import { SugarElement } from '../node/SugarElement';
+import * as SugarShadowDom from '../node/SugarShadowDom';
 import * as Html from '../properties/Html';
 import * as Replication from './Replication';
 
 const getHtml = (element: SugarElement<any>): string => {
```

Comment:
If you're going through and typing a bunch of sugar elements, should this one get updated also?

Reply:
I was just typing anything that didn't have a generic for code I touched. This one I believe would be `Node` but not 100% either, so I didn't both with it. Happy to change it if we're happy to put `Node` here.